rivus/gridder/extend_grid.py

```diff
@@ -6,21 +6,52 @@
 # Add Commodity columns filled with to the vertex DataFrame,
 def vert_init_commodities(vertex_df, commodities, sources=None):
     """Add commodity columns with zeros to vertex DataFrame
-        source: (Commodity, Index, Value) tuples/lists
-            e.g. ('Elec', 1, 2000)
+    source: (Commodity, Index, Value) tuples/lists
+        e.g. ('Elec', 1, 2000)
+
+    Args:
+        vertex_df ((Geo)DataFrame): vertex dataframe input
+        commodities (lis of str): Like ('Elec', 'Gas', 'Heat')
+        sources (list of tuples, optional): Init the source nodes
+            Like [('Elec', 0, 100000), ('Gas', 1, 50000)]
     """
     for commo in commodities:
         vertex_df[commo] = len(vertex_df) * [0]
     if sources:
         for s in sources:
             check1 = (len(s) == 3) and isinstance(s[0], str)
-            check2 = isinstance(s[1], int) and isinstance(s[2], (int, float))
+            check2 = isinstance(s[1], int) and isinstance(s[2], (int, float)) and \
```

Comment:
Instead of the flaky backslash, I would recommend using a `(`...`)` around the expression.